src/NHSD.BuyingCatalogue.Organisations.Api/Data/OrganisationEntityTypeConfiguration.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Newtonsoft.Json;
+using NHSD.BuyingCatalogue.Organisations.Api.Models;
+
+namespace NHSD.BuyingCatalogue.Organisations.Api.Data
+{
+    internal sealed class OrganisationEntityTypeConfiguration : IEntityTypeConfiguration<Organisation>
+    {
+        public void Configure(EntityTypeBuilder<Organisation> builder)
+        {
+            builder
+                .Property(entity => entity.Address)
+                .HasConversion(
+                    addressEntity => JsonConvert.SerializeObject(addressEntity, new JsonSerializerSettings
+                    {
+                        NullValueHandling = NullValueHandling.Ignore
```

Comment:
should we auto trim whitespace here too?

Reply:
Probably not necessary at the moment. The address data is taken from ODS & it not user configurable. This could go down as a tech debt ticket?